_i18n/de/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,53 @@
 
-After you have entered the 25 word mnemonic seed and have chosen your specific blockchain height, monero-wallet-cli will generate the same public address and view key as your old wallet and begin the refresh process automatically. (Please be patient as the refresh process may take a while.)
+Nachdem du die 25 Wörter und damit deinen mnemonischen Seed eingegeben und deine konkrete Blockchainhöhe gewählt hast, wird monero-wallet-cli dieselbe öffentliche Adresse und denselben View-Key deines alten Wallets generieren und den Aktualisierungsvorgang automatisch starten. (Bitte gedulde dich ein wenig; der Aktualisierungsvorgang kann eine Weile dauern.)
 
-### Account Software:  monero-wallet-gui
+### Konto-Software:  monero-wallet-gui
 
-Lauch `monero-wallet-gui`. If this is the first time you launched it go to the next step, otherwise click `Cancel`:
+Starte `monero-wallet-gui`. Gehe über zum nächsten Schritt, falls du das Programm zum ersten Mal startest, ansonsten wähle `Abbrechen`:
 
 ![cancel opening](png/restore_account/cancel-opening.png)
 
-Select your appropriate language `English(US)`:
+Wähle die für dich passende Sprache `Deutsch`:
 
 ![choose language](png/restore_account/choose-language.png)
 
-Click on `Restore wallet from keys or mnemonic seed`:
+Klicke auf `Stelle Wallet mit Schlüsseln oder mnemonischem Seed wieder her`:
 
 ![choose restore](png/restore_account/choose-restore.png)
 
-Keep `Restore from seed` selected, give your wallet a name & storage location, and complete the `Enter your 25 (or 24) word mnemonic seed`. Optionaly, specify a `Restore height (optional)` to avoid scanning of oldest blocks. Then click the `Right` arrow:
+Lasse `Mit mnemonischem Seed wiederherstellen` ausgewählt, lege Namen und Speicherstelle deines Wallets fest und vervollständige den Teil `Gebe deinen, aus 25 (oder 24) Wörtern bestehenden, mnemonischen Seed ein`. Um das Scannen der ältesten Blöcke zu umgehen, kannst du eine optionale `Wiederherstellungshöhe` festlegen. Klicke dann auf den `rechten` Pfeil:
 
 ![restore wallet](png/restore_account/restore-wallet.png)
 
-On the next page, give your wallet a strong password and confirm it before clicking the `Right` arrow:
+Auf der nächsten Seite gibst du deinem Wallet ein starkes Passwort. Bestätige dieses, bevor du erneut auf den `rechten` Pfeil klickst:
 
 ![wallet password](png/restore_account/wallet-password.png)
 
-Specify your daemon settings and click the `Right` arrow:
+Lege die Einstellungen deines Hintergrunddienstes fest und klicke abermals auf den `rechten` Pfeil:
 
 ![daemon settings](png/restore_account/daemon-settings.png)
 
-Click on `USE MONERO` to enjoy your restored wallet:
+Klicke auf `USE MONERO`, um dich mit deinem wiederhergestellten Wallet auszutoben:
```

Comment:
```suggestion
Klicke auf `USE MONERO` und hab Spaß / Freude mit deinem wiederhergestellten Wallet:
```

"sich mit einem Wallet austoben" sounds naughty :)

Reply:
Changed to "hab Spaß mit" (though I'm not sure if that sounds less naughty :D)